<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DListElementPanel</class>
 <widget class="QWidget" name="DListElementPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DListElementPanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="FindText_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Find:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FindInput_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="FindResult_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SearchUpward_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Search upward</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#SearchUpward_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/up_icon_20px.png);
background-position: center ;
padding-bottom:0;

}
#SearchUpward_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#SearchUpward_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/up_icon_20px.png);
background-position: center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SearchDownward_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Search downward</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#SearchDownward_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/down_icon_20px.png);
background-position: center ;
padding-bottom:0;

}
#SearchDownward_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#SearchDownward_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/down_icon_20px.png);
background-position: center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Button_horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="treewidget_horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="Buttons_verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="ZoomIn_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Zoom into the location of selected items</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">#ZoomIn_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/zoomin_20px.png);
background-position: center ;
padding-bottom:0;

}
#ZoomIn_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#ZoomIn_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/zoomin_20px.png);
background-position: center;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SelectAll_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select all expanded items</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#SelectAll_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/select_icon_20px.png);
background-position: center ;
padding-bottom:0;

}
#SelectAll_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#SelectAll_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/select_icon_20px.png);
background-position: center;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UnselectAll_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Unselected all items</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#UnselectAll_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/Unselect_icon_20px.png);
background-position: center ;
padding-bottom:0;

}
#UnselectAll_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#UnselectAll_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/Unselect_icon_20px.png);
background-position: center;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExpandAll_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Expand all elements</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#ExpandAll_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/expand_20px.png);
background-position: center ;
padding-bottom:0;

}
#ExpandAll_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#ExpandAll_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/expand_20px.png);
background-position: center;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CollapseAll_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Collapse all elements</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#CollapseAll_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/collapse_20px.png);
background-position: center ;
padding-bottom:0;

}
#CollapseAll_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#CollapseAll_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/collapse_20px.png);
background-position: center;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Close_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Close this dialog</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#Close_pushButton{
padding: 0px 0 0px 0;
background:none;
border-width: 2px;
border-style: solid;
border-radius: 2px;
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/cancel_20px.png);
background-position: center ;
padding-bottom:0;

}
#Close_pushButton:hover
{
background-color:rgb(30, 190, 220,  10%);
border-width: 2px;
border-style: solid;
border-color: rgb(29, 190, 220,  15%);
background-repeat:none;
}
#Close_pushButton:pressed
{
background:rgb(25, 185, 215,  20%);
border-width: 2px;
border-style: solid;
border-color:rgb(24, 185, 215,  25%);
background-repeat:none;
font: bold italic large &quot;Times New Roman&quot;;
text-align: bottom;
background-image: url(./icons/cancel_20px.png);
background-position: center;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <column>
        <property name="text">
         <string>Item name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Length</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Number of lane</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Free flow speed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Backward shockwave speed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Jam density</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Saturation Flow </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Demand/Sink</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Signalized</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Box Junction</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Junctions</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Junction Links</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Demand Sink Cells</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cells</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Arcs</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
